nautobot/extras/models/relationships.py

```diff
@@ -738,6 +735,9 @@ def _get_genericforeignkey(self, name):
 
         return None
 
+    def get_absolute_url(self):
+        self.relationship.get_absolute_url()
```

Comment:
```suggestion
        return self.relationship.get_absolute_url()
```

Reply:
I'm wondering if that was what was causing https://github.com/nautobot/nautobot/pull/3632#discussion_r1173202511 to still fail...